### PR TITLE
added a 'rollback_to' task to rollback to a particular version# along with all in-between

### PR DESCRIPTION
**Pull request by abionic**
added a 'rollback_to' task to rollback to a particular version# along with all in-between, as 'down' just undo a particular version and 'rollback' takes steps

actually I was looking for it during a project but due to its non-availability, wrote a small wrapper around rollback..... thought might help others looking for similar functionality

### CODE REVIEW

**Comment by adacosta** [comment]
If you or someone else can again verify functionality, I'll go ahead and merge.

File: lib/mongoid_rails_migrations/active_record_ext/migrations.rb [hunk 1]
```diff
@@ -195,6 +195,13 @@ def rollback(migrations_path, steps=1)
         move(:down, migrations_path, steps)
       end
 
+      def rollback_to(migrations_path, target_version)
+        all_versions = get_all_versions
+	rollback_to = all_versions.index(target_version.to_i) + 1
```
Comment (guille-moe) [inline lib/mongoid_rails_migrations/active_record_ext/migrations.rb]: Indents here looks a bit broken ^^' Everything else seems good !

Comment (jarthod) [reply lib/mongoid_rails_migrations/active_record_ext/migrations.rb]: Merged in master, I fixed the indentation and added a validation on `VERSION` presence

**Comment by jarthod** [comment]
Merged in `master`, I fixed the indentation and added a validation on `VERSION` presence

**Pull request merged**